app/src/main/java/com/onelist/dto/Item.kt

```diff
@@ -12,7 +12,14 @@ import com.google.gson.annotations.SerializedName
  * @property purchased boolean value to determine if the item was purchased or not
  * @constructor Creates an item with ID, name, category ID, quantity, and purchased value
  */
-data class Item(@SerializedName("id") var itemID: String = "", var name: String, var categoryIDs: List<String>, var quantity: Int, var purchased: Boolean) {
```

Comment:
When you have longer classes with many parameters, put them on different lines to increase readability. Especially when it gets too long on one line and you can't read it. https://kotlinlang.org/docs/coding-conventions.html#class-headers

Reply:
Formatting can, and should, be performed automatically, and consistently, by the IDE.  Formatting code is really programmer's preference anyway, and doesn't necessarily decrease technical debt.